src/app/shared/components/navbar/navbar.component.html

```diff
@@ -12,57 +12,74 @@
-  >
-  <a class="navbar__item" mat-button [routerLink]="[mainPageEnum.Maps]" routerLinkActive="navbar__item--active" ariaCurrentWhenActive="page"
-    >GIS-Browser</a
-  >
-  <a class="navbar__item" mat-button [routerLink]="[mainPageEnum.Data]" routerLinkActive="navbar__item--active" ariaCurrentWhenActive="page"
-    >Geodatenkatalog</a
-  >
-  <a
-    class="navbar__item"
-    mat-button
-    [routerLink]="[mainPageEnum.Support]"
-    routerLinkActive="navbar__item--active"
-    ariaCurrentWhenActive="page"
-    >Hilfe & Support</a
-  >
-  <a class="navbar__item" href="https://geodatenshop.zh.ch" mat-button target="_blank" rel="noopener noreferrer">
-    <mat-icon>launch</mat-icon>
-    Geodatenshop
-  </a>
-  <mat-divider class="navbar__divider" vertical></mat-divider>
-  @if (!this.isAuthenticated) {
-    <button (click)="this.startLogin()" class="navbar__item navbar__item--button" mat-button>Login</button>
-  } @else {
-    @if (screenMode === 'regular') {
-      <button [matMenuTriggerFor]="userMenu" class="navbar__item navbar__item--button" mat-button>
-        <mat-icon svgIcon="ktzh_user"></mat-icon>
-        {{ this.userName }}
+  @if (screenMode === 'regular') {
+    <a
+      [queryParams]="{}"
+      [routerLinkActiveOptions]="{exact: true}"
+      class="navbar__item"
+      mat-button
+      [routerLink]="[mainPageEnum.Start]"
+      routerLinkActive="navbar__item--active"
+      ariaCurrentWhenActive="page"
+      >Geoportal</a
+    >
+    <a
+      class="navbar__item"
+      mat-button
+      [routerLink]="[mainPageEnum.Maps]"
+      routerLinkActive="navbar__item--active"
+      ariaCurrentWhenActive="page"
+      >GIS-Browser</a
+    >
+    <a
+      class="navbar__item"
+      mat-button
+      [routerLink]="[mainPageEnum.Data]"
+      routerLinkActive="navbar__item--active"
+      ariaCurrentWhenActive="page"
+      >Geodatenkatalog</a
+    >
+    <a
+      class="navbar__item"
+      mat-button
+      [routerLink]="[mainPageEnum.Support]"
+      routerLinkActive="navbar__item--active"
+      ariaCurrentWhenActive="page"
+      >Hilfe & Support</a
+    >
+    <a class="navbar__item" href="https://geodatenshop.zh.ch" mat-button target="_blank" rel="noopener noreferrer">
+      <mat-icon>launch</mat-icon>
+      Geodatenshop
+    </a>
+    <mat-divider class="navbar__divider" vertical></mat-divider>
+    @if (!this.isAuthenticated) {
+      <button (click)="this.startLogin()" class="navbar__item navbar__item--button" mat-button>
+        <mat-icon svgIcon="ktzh_user"></mat-icon>Login
       </button>
     } @else {
-      <button [matMenuTriggerFor]="userMenu" class="navbar__item navbar__item--button" mat-button #trigger="matMenuTrigger">
+      <button [matMenuTriggerFor]="userMenu" class="navbar__item navbar__item--button" mat-button>
         <mat-icon svgIcon="ktzh_user"></mat-icon>
-        <mat-icon [fontIcon]="trigger.menuOpen ? 'arrow_drop_up' : 'arrow_drop_down'"></mat-icon>
+        {{ this.userName }}
```

Comment:
unnecessary `this`

**Edit:** there are a couple more unnecessary `this` within this template